app/notification.go

```diff
@@ -266,15 +266,15 @@ func (a *App) SendNotifications(post *model.Post, team *model.Team, channel *mod
 	// MUST be completed before push notifications send
 	for _, uchan := range updateMentionChans {
 		if result := <-uchan; result.Err != nil {
-			l4g.Warn(utils.T("api.post.update_mention_count_and_forget.update_error"), post.Id, post.ChannelId, result.Err)
+			mlog.Warn(fmt.Sprintf("Failed to update mention count, post_id=%v channel_id=%v err=%v", post.Id, post.ChannelId, result.Err), mlog.String("postid", post.Id))
```

Comment:
Instead of `postid` perhaps `postId` or `post_id`? It would be nice for these to remain human-readable even after getting ingested.

Reply:
Ya I agree.